spring-data-querydsl-predicate-injector/build.gradle

```
@@ -0,0 +1,42 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+dependencies {
+    internalPlatform platform(project(':thunx-dependencies'))
+
+    compileOnly 'org.projectlombok:lombok'
+    annotationProcessor 'org.projectlombok:lombok'
+
+    api 'com.querydsl:querydsl-core'
+    api 'org.springframework:spring-core'
+
+    implementation 'org.springframework.data:spring-data-rest-webmvc'
+
+    compileOnly "org.springframework.boot:spring-boot-autoconfigure" // for autoconfiguration annotations
```

Is this appropriate moment to use `because` ?
```suggestion
    compileOnly("org.springframework.boot:spring-boot-autoconfigure") {
        because 'needed for autoconfiguration annotations'
    }
```

> Returns:
>   a reason to use this dependency

---

I was actually marking it that it is compileOnly because we *only* need it for autoconfiguration annotations